lib/services/local/access_list.go

```diff
@@ -129,6 +129,11 @@ func (a *AccessListService) GetAccessList(ctx context.Context, name string) (*ac
 	return accessList, trace.Wrap(err)
 }
 
+// GetAccessListsToReview returns access lists that the user needs to review. This is not implemented in the local service.
+func (a *AccessListService) GetAccessListsToReview(ctx context.Context) ([]*accesslist.AccessList, error) {
+	return nil, trace.NotImplemented("GetAccessListsToReview should not be called")
```

Comment:
I may be missing something but how does it work if the backend method is not implemented?

Reply:
It's only implemented at the gRPC service level -- basically, the backend can't do this alone because it doesn't have the auth context needed. Promotion does largely the same thing here.

Reply:
Here's the corresponding `e` PR that implements this service side: https://github.com/gravitational/teleport.e/pull/2401